SCons/Taskmaster/__init__.py

```diff
@@ -735,14 +789,13 @@ def _validate_pending_children(self):
             for p in n.waiting_parents:
                 assert p.ref_count > 0, (str(n), str(p), p.ref_count)
 
+    def tm_trace_message(self, message):
+        return 'Taskmaster: %s' % message
```

Comment:
this is going to get you lint complaints (if tool smart enough to follow through a function call) - logging world suggests using "lazy interpolation".

Reply:
I guess I got rid of all usages of this so I've deleted it.

Reply:
Removed this function as I actually removed all calls to it.. ;)